guestbook-archetype/pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.google.appengine.archetypes</groupId>
   <artifactId>guestbook-archetype</artifactId>
-  <version>2.0.0-1.9.10</version>
+  <version>2.0.0-1.9.20</version>
```

Comment:
Moving to objectify is a major change, how about to change the version to 3.0.0-1.9.20 ?